doc/examples/petab.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PEtab import"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This illustrates how the PEtab format can be used together with the ODE simulation toolbox AMICI to define ODE based parameter estimation problems for pyABC.\n",
+    "\n",
+    "To use this functionality, you need to have (at least) PEtab and AMICI installed. You can obtain these by installing pyABC with\n",
```

Comment:
Experience tells, it's good to directly link to the installation instructions ;)